radis/lbl/factory.py

```diff
@@ -542,7 +542,13 @@ def __init__(
 
         if truncation == 0:
             raise ValueError(
-                "Lineshape truncation must be >0. If you want no truncation (compute lineshape on the full spectral range), use `truncation=None`"
+                "Lineshape truncation must be > 0. If you want no truncation (compute lineshape on the full spectral range), use `truncation=None`"
```

Comment:
Can you also add :

"Note (advanced) : no truncation is not physically more accurate. Most molecules exhibit a sub-lorentzian behavior far from the line centers. A truncation at around 40-50 cm-1 is a good choice"